guest/api/proto/proto.go

```diff
@@ -22,6 +22,11 @@ import (
 	meta "sigs.k8s.io/kube-scheduler-wasm-extension/kubernetes/proto/meta"
 )
 
+type Nodes interface {
```

Comment:
thinking about exposing the slice like this as there's no lazy slice in Go. Will try it out ergonomically once finished other stuff.